integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -24,20 +37,30 @@ export class RasaNlu extends NluPlugin<RasaNluConfig> {
     return {
       serverUrl: 'http://localhost:5005',
       serverPath: '/model/parse',
+      alternativeIntents: { maxAlternatives: 15, confidenceCutoff: 0.0 },
     };
   }
 
-  async process(handleRequest: HandleRequest, jovo: Jovo): Promise<NluData | undefined> {
+  async process(handleRequest: HandleRequest, jovo: Jovo): Promise<RasaNluData | undefined> {
     const text = jovo.$request.getRawText();
     if (!text) return;
     try {
       const rasaResponse = await this.sendTextToRasaServer(text || '');
```

Comment:
Alternative value of `''` is not needed due to checking if the text exists before.

Reply:
Done